packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -558,36 +557,76 @@ export function getEnvForPackageManager({
     ...env,
   };
 
-  if (newPath) {
+  const alreadyInPath = (path: string) => {
+    const oldPath = env.PATH ?? '';
+    return oldPath.includes(path);
```

Comment:
We should probably split the PATH into an array, otherwise this function might have a false positive where a substring is matched (contrived example: `alreadyInPath('/usr')` where `PATH` is something like `/usr/local/bin:/bin`):

```suggestion
    return oldPath.split(path.sep).includes(path);
```